.npmignore

```diff
@@ -1,2 +1,13 @@
 test/
-tmp/
\ No newline at end of file
+tmp/
+scripts/
```

Comment:
@nfischer: I suppose we could, but I can't think of a reason anyone who downloads it from npm would ever need them, so I thought we may as well save them the bytes.